dev-ruby/net-ssh/net-ssh-6.2.0-r1.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+USE_RUBY="ruby24 ruby25 ruby26 ruby27"
+
+RUBY_FAKEGEM_DOCDIR="doc"
+RUBY_FAKEGEM_EXTRADOC="CHANGES.txt README.md THANKS.txt"
+RUBY_FAKEGEM_EXTRAINSTALL="support"
+
+inherit ruby-fakegem
+
+DESCRIPTION="Non-interactive SSH processing in pure Ruby"
+HOMEPAGE="https://github.com/net-ssh/net-ssh"
+SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.rc1.tar.gz -> net-ssh-git-${PV}.tgz"
+
+LICENSE="GPL-2"
+SLOT="$(ver_cut 1)"
+KEYWORDS="amd64 ~arm ppc ppc64 x86"
```

Comment:
KEYWORDS need ~ after a version bump run ekeyword ~all YOUREBUILD

Reply:
@jonasstein Hey,
I have upgrade the description of the package with the one given on rubygems.org
also fixed the keywords of dev-ruby/net-ssh

Please tell me if you see something else to fix.

Thank you